pullrequest.js

```diff
@@ -77,6 +93,25 @@ chrome.storage.sync.get({url: ''}, function(items) {
 
         $body.on('click', '.js-collapse-deletions', collapseDeletions);
 
+        if (items.tabSwitchingEnabled) {
```

Comment:
You'd have to reload the page you're on to get this to re-read. We could change that by adding a listener to the chrome storage, kinda like we have on the options page, that would read the settings and enable or disable this accordingly. Might be too much work though, just to have it work on the fly (something that probably isn't a big deal since it starts off).